Dockerfile

```diff
@@ -3,35 +3,23 @@
 #
 # http://github.com/aerospike/aerospike-tools.docker
 #
-FROM debian:bullseye-slim AS build
+FROM debian:bookworm-slim
 
 ARG TARGETARCH
 
 RUN \
   apt-get update -y \
   && apt-get install -y \
   wget \
-  build-essential \
-  zlib1g-dev \
-  libncurses5-dev \
-  libgdbm-dev \
-  libnss3-dev \
-  libssl-dev \
-  libreadline-dev \
-  libffi-dev \
-  libsqlite3-dev \
-  libbz2-dev
-
-# Build and Install Python from source
-RUN \
-  wget https://www.python.org/ftp/python/3.10.9/Python-3.10.9.tgz \
-  && tar -xf Python-*.tgz \
-  && cd Python-*/ \
-  && ./configure --enable-optimizations \
-  && make
+  python3 \
+  libreadline8
```

Comment:
First install of libreadline. (Probably not needed in a multi-stage build)

Reply:
added back multi stage build, this is only done in the install stage now